iosApp/PhongKMMIC/Sources/Presentation/Modules/SurveyQuestions/View/Answer/NpsAnswerView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  NpsAnswerView.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 16/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct NpsAnswerView: View {
+
+    @State private var selectedIndex: Int?
+
+    var body: some View {
+        VStack(spacing: 16.0) {
+            answersView
+            legendView
+        }
+    }
+
+    private var answersView: some View {
+        HStack(spacing: 0.0) {
+            ForEach(0 ..< 10, id: \.self) { index in
```

Comment:
what do you think about adding a private constant for this `10` value? 🙏

Reply:
This is a dummy value for testing, I will replace it with actual value in the integration task